app/classifier/components/Intervention/Intervention.jsx

```diff
@@ -14,14 +14,15 @@ const StyledInterventionMessage = styled.div`
   }
 `;
 
-function Intervention({ onUnmount, intervention, user }) {
+function Intervention(props) {
+  const { onUnmount, intervention, user } = props;
   const { message } = intervention;
   const checkbox = React.createRef();
 
   useEffect(() => {
     // the return value of an effect will be called to clean up after the component
     return onUnmount;
-  });
+  }, []);
```

Comment:
OOI - what does this empty array do, signal what props are used to determine when the `onUnmount` function is called vs when the `useEffect` hook is run?

Should we add a comment explaining that?

Reply:
https://github.com/zooniverse/Panoptes-Front-End/pull/5341#discussion_r284258363